### DISCUSSION TITLE
Supporting a button for disabling auto-play videos in webpages

### DISCUSSION THREAD

**Discussion by CC-Hsu** [plateaukao/einkbro#371] [Ideas]
Hi, Team,

Most E-ink screens are poor at handling videos playing.

But sometimes some videos in some web pages are played automatically.

It would be great if there is a button for disable auto-play videos.

Best Regards,

**Comment by plateaukao** [comment]
@CC-Hsu 
give me some websites that will auto play videos, so that I can verify if my fix works.

**Comment by CC-Hsu** [comment]
Hi, plateaukao,

Thanks a lot for this feature improvement.

For auto-play web pages, in the CNN homepage there are some videos that are auto-played.

https://edition.cnn.com/

And some pages will contains memes GIF files; this can be found by searching "memes GIF" in search engines.

Best Regards.
